README.md

```diff
@@ -157,4 +157,17 @@ In a few instances, the creation pattern of a resource type doesn't follow norma
 
 There is currently no plan to change this behavior. If this scenario is important to you, please [open a support ticket](https://azure.microsoft.com/support/create-ticket/) with the Azure SQL team.
 
+### Provider pass-through to non Azure Resource Manager resources
+
+There are examples where a resource provider publishes a resource type to Azure Resource Manager, but the resources it represents cannot be managed by Azure Resource Manager. For example, Microsoft.Web has published several resource types to Azure Resource Manager that actually represent resources of the customer's site rather than Azure Resource Manager resources. Such resources cannot or should not be managed by Azure policy, and are explicitly excluded. All known examples are listed here:
```

Comment:
but the resource' _s_ ' it represents cannot be
Should the 's' be removed? (I don't know actually, just in case you missed)

Reply:
Thanks for checking. Either with or without the s is grammatically correct, but with the s captures the meaning better, since leaving it off implies that a resource type can only represent one resource instance. Since resource types represent all instances of their type, the plural is better.